localgov_theme_starterkit/localgov_theme.theme

```diff
@@ -0,0 +1,165 @@
+<?php
```

Comment:
Are files like these meant to be here? Would it cause issue with the other theme files?